frontend/amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -160,7 +162,7 @@ def create_issue(self,
             }, labels=self.issue_labels,
                 summary=title,
                 description=(f'{description} '
-                             f'\n *Reported By:* {user_email} '
+                             f'\n *Reported By:* {reporting_user_str if reporting_user_str else user_email} '
```

Comment:
This is a small unrelated change to follow up on previous frontend Jira work. This will link the profile url for the user who created the Jira ticket rather than only displaying their email.